travis/install.sh

```diff
@@ -19,6 +19,7 @@ case "$SUITE" in
         . "$HOME/.nvm/nvm.sh"
         nvm install
         nvm use
+        nvm alias default $(cat .nvmrc)
```

Comment:
This was an attempt to use the selected Node.js version from `.nvmrc`, but somehow the `npm run stylelint` still fails claiming it's using "v0.10.36" instead of "v6.1" that is defined in the `.nvmrc` file.